demo.yml

```diff
@@ -0,0 +1,3 @@
+requirements:
```

Comment:
Does this file get magically used somehow? I don't see anywhere here using it.

If so, can we update it for `jsonschema 3`? I think everything above looks like it shouldn't require any changes to do so.

Reply:
The `demo.yml` file is automatically used by notebooks.ai while loading the Docker container. It's basically loading a base container Image and then installing all Python packages under "requirements".

I updated `jsonschema` to version 3.0.1.

Reply:
Got it -- and is it supposed to be the equivalent of a `requirements.txt` then?

Because you have `pyrsistent` here, but that's not the only dependency (or transitive dependency) of `jsonschema`.

Reply:
I also don't really see any docs linked anywhere on https://notebooks.ai -- where do they keep those?

Reply:
Hey @martinzugnoni! Polite ping on the above, I know life has gotten in the way for both of us :) but yeah curious to hear what you think.

Reply:
Hey @Julian, I'm very sorry about the late response. I've been super busy lately. 

Requirements listed in the `demo.yml` file are the libraries needed to run the `DEMO.ipynb` notebook. It seems that just by installing `jsonschema` the DEMO is not fully working (see screenshot below). That's why I also added `pyrsistent`.

![image](https://user-images.githubusercontent.com/1155573/56437149-ba417c80-62b4-11e9-8f2a-195f0af73cd2.png)

About notebooks.ai documentation, we are still working on it. We will probably link the documentation to the main website once it's done.